samples/msal-browser-samples/HybridSample/routes/auth.js

```diff
@@ -48,27 +55,28 @@ router.post('/server-redirect', (req, res) => {
     // Parameters needed for hybrid spa test flight
     tokenRequest.tokenQueryParameters = {
         dc: "ESTS-PUB-WUS2-AZ1-FD000-TEST1",
-        hybridspa: "true"
+        hybridspa: "true",
     }
 
     // If using hybrid spa flow, enable feature flag to get additional auth code
     if (useHybrid) {
         console.log('Hybrid enabled');
-        tokenRequest.enableSpaAuthorizationCode = true
-        // tokenRequest.tokenBodyParameters = {
-        //     return_spa_code: "1"
-        // }
+        tokenRequest.enableSpaAuthorizationCode = true;
+        tokenRequest.tokenBodyParameters = {
+            return_spa_code: "1",
+            "webnativebroker": "true"
```

Comment:
Don't think this is needed anymore and if it is it should be "webnativebridge" right?

Reply:
In test slice we do not, in prod we do. Atleast that was my understanding.

Reply:
In any case, the param isn't correct. Was this tested? If it worked that means this could probably be removed...

Reply:
@lalimasharda  to confirm it.

Reply:
It is working without as well so I have removed it